diff --git a/dist/queries.js b/dist/queries.js
index 854fbcd226aad285eacbcd2bd540170e2f0ec104..3b2d62d40c3fa91f4f71ff1c0f9cef9462bd50fb 100644
--- a/dist/queries.js
+++ b/dist/queries.js
@@ -113,6 +113,7 @@ async function getMatches(root) {
     return edgedb_1.adapter.walk(root, {
         match: [/[^\/]\.edgeql$/],
         skip: [
+            /\.(?:git|svelte-kit)/,
             /node_modules/,
             RegExp(`dbschema\\${edgedb_1.adapter.path.sep}migrations`),
             RegExp(`dbschema\\${edgedb_1.adapter.path.sep}fixups`),
