diff --git a/dist/queries.js b/dist/queries.js
index 3ebf44a7cbcc789be81b64fe311cf3551d113cfb..afd42c580a2f8a38c8c3509f087f645af4c23887 100644
--- a/dist/queries.js
+++ b/dist/queries.js
@@ -111,7 +111,7 @@ async function getMatches(root, schemaDir) {
     return edgedb_1.adapter.walk(root, {
         match: [/[^/]\.edgeql$/],
         skip: [
-            /node_modules/,
+            /\.(?:git|svelte-kit)|node_modules|build/,
             RegExp(`${schemaDir}\\${edgedb_1.adapter.path.sep}migrations`),
             RegExp(`${schemaDir}\\${edgedb_1.adapter.path.sep}fixups`),
         ],
